src/mapengine/leaflet/js/leaflet-map-engine.debug.js

```diff
@@ -650,26 +661,24 @@ emp.engineDefs.leafletMapEngine = function (args) {
         instanceInterface.addEmpObject(oControl);
         instanceInterface.leafletInstance.addControl(oControl);
 
-        oControl = new L.Control.Zoom({
-            position: 'topright',
-            objectType: leafLet.typeLibrary.objectType.LL_CONTROL,
-            coreId: emp.helpers.id.newGUID()
-        });
-
-        instanceInterface.oZoomControl = oControl;
-
-        oLayerEntry = new emp.typeLibrary.Static({
-            name: 'Zoom Buttons',
-            featureId: oControl.getCoreId(),
-            overlayId: sCoreId,
-            visible: true,
-            coreId: oControl.getCoreId(),
-            coreParent: sCoreId
-        });
-        oLayerList.push(oLayerEntry);
-
-        instanceInterface.addEmpObject(oControl);
-        instanceInterface.leafletInstance.addControl(oControl);
+        // oControl = new L.Control.Zoom({
```

Comment:
Once Jimmy tests the default zoom control that is being used I will remove this code.